src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return ((double) Math.round(Math.PI * radius * radius * 10) / 10);
```

Comment:
All magic numbers in your code should be constants. It's a good practice to define '10' as a constant with a descriptive name to explain what it represents.